tests/testthat/helper-get_join_keys.R

```diff
@@ -1,4 +1,4 @@
-#' Generate a teal_data dataset with sample data and JoinKeys
+#' Generate a teal_data dataset with sample data and join_keys
```

Comment:
Please move all helpers to `R/testthat-helpers.R`

https://community.rstudio.com/t/why-are-tests-testthat-helper-files-discouraged-in-testthat/85253

Reply:
I don't think recommendation is still valid, as `{testthat}` docs changed a bit since 2020

New (7 months old) vignette: https://testthat.r-lib.org/articles/special-files.html

_(emphasis mine as the helpers functions are all about redundant code -- except one)_

> Helper files live in tests/testtthat/, start with helper, and end with .r or .R. They are sourced by devtools::load_all() (so they’re available interactively when developing your packages) and by [test_check()](https://testthat.r-lib.org/reference/test_package.html) and friends (so that they’re available no matter how your tests are executed).
>
> **Helper files are a useful place for functions that you’ve extracted from repeated code in your tests**, whether that be test fixtures ([vignette("test-fixtures")](https://testthat.r-lib.org/articles/test-fixtures.html)), custom expectations (vignette("custom-expectations")), or skip helpers ([vignette("skipping")](https://testthat.r-lib.org/articles/skipping.html)).

The exception: `compare.join_keys` will live better in `R/compare.R`, however I'd argue that we should use testthat 3rd edition on `{teal.data}` and discard the need for this with as `{waldo}` compare framework is used (that tests for unordered lists).

Reply:
Actually, `compare.join_keys` cannot be in `R/testthat-helpers.R` as it depends on a S3 method available on `testthat` (which is only a `Suggests`

Reply:
Thanks, let's follow most recent recommendations 👍